### DISCUSSION TITLE
R2modman opening java error

### DISCUSSION THREAD

**Discussion by zumrok11** [ebkr/r2modmanPlus#815] [Q&A]
![r2](https://user-images.githubusercontent.com/109094777/178304800-fe4120f6-15d1-4b7a-9cb1-fa2f85f2049e.png)

anyone knows solution?

**Comment by ebkr** [comment]
Use the portable version of 3.1.29 in the GitHub releases for now. Latest update broke support for win 7.

**Reply by eightisgreat** [reply]
Getting the older version works, except if you ever have to close the application and then it auto-updates.
